vtex-apps/checkout-payment
# Add payment list and installment options for credit card

#### What problem is this solving?

This PR adds the payment options list section before showing the card form, and also adds the installment options selection after filling the card details. This also adds the summary for the payment data in the review step.

#### How should this be manually tested?

[Workspace](https://lucas--checkoutio.myvtex.com/cart/add?sku=289).

Test plan:

1. Proceed to the payment step in checkout, filling the previous steps accordingly.
2. Click on "New Credit Card" in the payment option list.
3. Fill out the credit card form (you can use the same fake credit card we use internally).
4. Select an installment option.
5. You should be redirected to the review page, with the correct details in the payment summary, and the place order button should be visible.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable)](https://app.clubhouse.io/vtex/story/34723/adicionar-etapa-de-parcelamento-no-checkout-payment).

#### Screenshots or example usage

Payment summary:
![image](https://user-images.githubusercontent.com/10223856/81703532-1dafc680-9443-11ea-911f-14cf1e120ca2.png)

Payment option list:
![image](https://user-images.githubusercontent.com/10223856/81703560-27d1c500-9443-11ea-9c05-481d30feef52.png)

Credit card form:
![image](https://user-images.githubusercontent.com/10223856/81703581-2dc7a600-9443-11ea-8418-d58ccde3dfd1.png)

Installment options list:
![image](https://user-images.githubusercontent.com/10223856/81703642-3fa94900-9443-11ea-88ce-8d9c5f62d6bb.png)

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
